tests/test-cases/Group6-VIC-Machine/6-03-Delete.md

```diff
@@ -36,3 +36,13 @@ This test requires that a vSphere server is running and available
 ### Expected Outcome:
 1. All steps should succeed
 2. govc should return 1 when not finding images in step 6
+
+## Delete VCH with non-cVM in same RP
+1. Create VCH though vic-machine create
+2. Create VM in the VCH's resource pool
+3. Delete VCH
+4. Check vSphere through govc to make sure the VM was not deleted
+5. Delete the VM and RP to cleanup
+
+### Expected Outcome:
+1. All steps should succeed
```

Comment:
We need another two tests:
1. endpointVM is moved out of it's pool, pool is left alone
2. endpointVM is moved into the root of the cluster and the original pool is deleted.

Reply:
Just to be clear, for 1) the endpointVM should be successfully deleted and the RP and its children should not be deleted
and for 2) the original pool should be deleted OOB after endpointVM is moved to the root pool then vic-machine delete should successfully delete the endpointVM?
Thanks

Reply:
1) correct
2) correct - and other VMs in the root pool should be left alone (which is what the current test asserts but worth being explicit)